core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -17895,4 +17895,27 @@ private int resolveGroupListLimit(int givenLimit) {
         }
         return givenLimit;
     }
+
+    /**
+     * Update group name in the hybrid group-role mapper.
+     *
+     * @param groupName        The current group name.
+     * @param newGroupName     The new group name.
+     * @throws UserStoreException An unexpected exception has occurred.
+     */
+    public void updateHybridGroupName(String groupName, String newGroupName) throws UserStoreException {
```

Comment:
why HybridGroup ?

Reply:
is this related groups or roles?

Reply:
Related to groups, update the group name in the UM_HYBRID_GROUP_ROLE table, will change the method name.